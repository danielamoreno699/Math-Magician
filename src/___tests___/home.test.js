import { render, screen } from '@testing-library/react';
import Home from '../components/home';

describe('must show content when the user navigates to the home page ', () => {
  test('when navigates to home page it displays the title ', () => {
    const { container } = render(<Home />);

    expect(screen.getByText('Welcome to our page')).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });
});
