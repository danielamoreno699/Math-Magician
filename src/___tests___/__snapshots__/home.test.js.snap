// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`must show content when the user navigates to the home page  when navigates to home page it displays the title  1`] = `
<div
  class="text mt-5"
>
  <h1>
    Welcome to our page
  </h1>
  <p
    class="p-one mt-4"
  >
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem,eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?
  </p>
  <p
    class="p-two mt-4"
  >
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat repudiandae dolor similique, nulla quam exercitationem, eveniet rem expedita aliquid excepturi dicta aspernatur. Tempora dolorem consequuntur vero minima maiores dicta at?
  </p>
</div>
`;
